contrib/automation_tests/orbit_symbol_cache.py

```diff
@@ -26,12 +27,13 @@
  - delete the symbol cache
  - load all symbols, store loading times, verify symbol files exist in cache
  - restart session
- - load all symbols again, loading times should have decreased significantly
+ - load all symbols again, loading times should have decreased. (This has a *very* high threshold for failure
```

Comment:
My gut feeling is that this the run time is just a random number generator and that this will still be unstable. But let's give it a try.